helpers/schema_compiler_test.go

```diff
@@ -0,0 +1,113 @@
+package helpers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/pb33f/libopenapi-validator/config"
+)
+
+// A few simple JSON Schemas
+const stringSchema = `{
+  "type": "string",
+  "format": "date",
+  "minLength": 10
+}`
+
+const objectSchema = `{
+  "type": "object",
+  "title" : "Fish",
+  "properties" : {
+     "name" : {
+	    "type": "string",
+        "description": "The given name of the fish"
+     },
+     "species" : {
+		"type" : "string",
+		"enum" : [ "OTHER", "GUPPY", "PIKE", "BASS" ]
+     }
+  }
+}`
+
+func Test_SchemaWithNilOptions(t *testing.T) {
+	jsch, err := NewCompiledSchema("test", []byte(stringSchema), nil)
+
+	require.Nil(t, err, "Failed to compile Schema")
```

Comment:
You misread my previous comment

I was asking to remove err as argument, but I was asking for this


```suggestion
	require.NoError(t, err, "Failed to compile Schema")
```

Reply:
Everyday is a learning experience ;-)

Reply:
I suggest you enabling

testifylint linter through golangci-lint